dhis2-android-app/src/main/java/org/dhis2/mobile/processors/OfflineDataProcessor.java

```diff
@@ -116,7 +116,7 @@ private static void uploadOfflineReports(Context context) {
                     NotificationBuilder.fireNotification(context,
```

Comment:
According to the issue description "Show sync errors as pop-ups instead of Android notifications" and you here when an error occurred send notification and show a dialog, we should not send notification only show a dialog.